rock_paper_scissors/rps.py

```diff
@@ -2,8 +2,10 @@
 
 import sys
 
+
 def rock_paper_scissors(n):
-  pass 
+  user = input("")
```

Comment:
Hahaha, remember to finish this when you have the opportunity. Not sure if you need to though... Hmm.